### PR TITLE
Joaquin/feat/associate and certificate

### PR DESCRIPTION
**Pull request by JKinnez**

### CODE REVIEW

File: app/views/students/_form.html.erb [hunk 1]
```diff
@@ -9,6 +9,14 @@
   <%= f.input :telephone %>
   <%= f.input :diseases %>
   <%= f.input :observations %>
+  <%= f.input :documents, input_html: { multiple: true, accept: ".pdf, .docx, .doc, .odt" } %>
+  <div class="flex justify-end mb-4">
+    <% @student.documents.each do |d| %>
```
Comment (mapreal19) [inline app/views/students/_form.html.erb:14]: don't use `d` for naming the variable. Use `document` instead.

File: config/environments/development.rb [hunk 2]
```diff
@@ -30,6 +30,8 @@
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
+  # Avoid replacing current files when updating with new files. https://github.com/rails/rails/issues/36374
+  config.active_storage.replace_on_assign_to_many = false
```
Comment (mapreal19) [inline config/environments/development.rb:34]: If this is the same for the 3 environments put this config in the application.rb file

**Review by mapreal19 — CHANGES_REQUESTED** [review]

File: app/views/students/_form.html.erb (hunk 1 above)
Comment (JKinnez) [reply app/views/students/_form.html.erb]: Done

File: config/environments/development.rb (hunk 2 above)
Comment (JKinnez) [reply config/environments/development.rb]: Done

**Review by mapreal19 — APPROVED** [review]

**Pull request merged**